### ISSUE TITLE
pkill shortcut for when-if mouse locked in drag-drop

### ISSUE THREAD

**Issue by cmrdt** [linuxmint/nemo#1520]
when doing drag&drop (not just linuxmint affects ubuntu also, 
but affected process might mitigate-workaround)
occasionally the mouse gets stuck in that state to some window 
& has mouse-focus only and/or mouse stuck in drag-mode (cursor)  
(2009 to now unresolved!)[Bug #402892 “Mouse cursor gets stuck in “drag and drop” mode ](https://bugs.launchpad.net/bugs/402892) 
 but if able, use keyboard to alt+tab to a terminal 
and pkill whatever process was in focus when whatever-buggy-framework(not sure which) glitched-locked-down

(restarting cinnamon does not help though)
if easier way to pkill if terminal window is not there..

**Comment by angelsix** [comment]
I get same issue on fresh install of Ubuntu 17 on late 2017 Dell XPS. 

Seemed to be from a drag-drop from desktop to a folder. Quick fix was to do `ps -a` and find the PID of the nautilus-desktop and then doing `kill pid`

**Comment by bbbbbr** [comment]
I've seen this on occasion as well. Quite annoying when it happens.  Seeing the same behavior that a restart of cinnamon doesn't clear the stuck Drag and Drop operation.

If a user wasn't familiar with the workaround it could leave them stuck and likely to just reboot.

**Comment by xenopeek** [comment]
@bbbbbr can you add some more detail under which circumstances you've seen this issue? I.e., does this happen dragging a very large file, very many files, between certain storage devices or anything other that can help developers to reproduce this?

If you're on Linux Mint, which version are you on and will you be upgrading to Linux Mint 20? It would be most useful to have hints how to reproduce this and confirmation as to whether this still affects Linux Mint 20.

**Comment by bbbbbr** [comment]
Yes, I'm on Linux Mint. I do plan to upgrade, but probably not for a couple months since I want to finish a development project before disturbing any of the system setup.

```$ cat /etc/linuxmint/info 
RELEASE=18.3 
CODENAME=sylvia 
EDITION="Cinnamon 64-bit" 
DESCRIPTION="Linux Mint 18.3 Sylvia" 
DESKTOP=Gnome 
TOOLKIT=GTK 
```

It's fairly reproducible when dragging ".mod" audio files into an existing instance of xplayer. It often happens after ~ 5-10 drag-and-drop actions. 

A .mod file in case you don't have one handy:
[Twilight Drive.mod.zip](https://github.com/linuxmint/nemo/files/4840764/Twilight.Drive.mod.zip)

I'm using a laptop with a ~9 year old processor (hyper threading turned off), in case this might be speed/timing related.

Thanks!

**Comment by damadden** [comment]
I experienced this just now.  My system is **RHEL 7.7** and I connect to it via TightVNC JViewer from a Windows 10 system.  I was able to get rid of the mouse "drag and drop" issue by pressing `Alt+Esc`.

On a side note, little did I realize that this also locked the `Alt` key in my VNC session, so all my typing was screwed up.  Once I realized and unlocked the `Alt`, everything was back to normal.
